draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -411,42 +411,45 @@ extension, that can improve the precision of the feedback from QUIC is
 {{!I-D.draft-ietf-quic-ack-frequency}}, which allows a sender to control the
 delay of acknowledgments sent by the receiver.
 
+The list of RTCP Receiver Reports that could be replaced by feedback from QUIC follows:
+
 * *Receiver Reports* (`PT=201`, `Name=RR`, {{!RFC3550}})
-  * *Fraction lost*: The fraction of lost packets can be directly infered from
+  * *Fraction lost*: When RTP packets are carried in QUIC datagrams, the fraction of lost packets can be directly inferred from
     QUIC's acknowledgments. The calculation SHOULD include all packets up to the
     acknowledged RTP packet with the highest RTP sequence number. Later packets
     SHOULD be ignored, since they may still be in flight, unless other QUIC
-    packets that were sent after the datagram frame, were already acknowledged.
+    packets that were sent after the RTP packet frame, were already acknowledged.
```

Comment:
```suggestion
    packets that were sent after the RTP packet, were already acknowledged.
```

In streams, the RTP packet can be spread over many frames.

Reply:
Fixed.